docs/earthfile/builtin-args.md

```diff
@@ -30,6 +30,7 @@ The following builtin args are available
 | `EARTHLY_GIT_AUTHOR` | The git author detected within the build context directory. If no git directory is detected, then the value is an empty string. | `John Doe <john@example.com>` |
 | `EARTHLY_GIT_CO_AUTHORS` | The git co-authors detected within the build context directory, separated by space. If no git directory is detected, then the value is an empty string. | `Jane Doe <jane@example.com Jack Smith <jack@example.com>` |
 | `EARTHLY_GIT_COMMIT_AUTHOR_TIMESTAMP` | The author timestamp, as unix seconds, of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `1626881847` |
+| `EARTHLY_GIT_BRANCH` | The git branch of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `main` |
```

Comment:
Yay! We can edit docs in the same PR!

BTW should we add an `*` or some mention that this is experimental and only available through a feature flag?

Reply:
good point, I added on a `Must be enabled with `VERSION --git-branch 0.7` feature flag.`